src/components/Fingerprint.vue

```diff
@@ -166,5 +168,21 @@ export default class Fingerprint extends ( mixins( Messages ) as VueConstructor<
 			@include fontSize( 23px );
 		}
 	}
+	.wikibase-termbox-fingerprint-list & {
+		@include media-breakpoint-up( md ) {
+			.wikibase-termbox-fingerprint__terms {
+				display: flex;
+				.wikibase-termbox-fingerprint__label,
+				.wikibase-termbox-fingerprint__description,
+				.wikibase-termbox-fingerprint__aliases {
+					flex: 1;
+				}
+				.wikibase-termbox-fingerprint__description,
+				.wikibase-termbox-fingerprint__aliases {
+					margin-left: 16px;
```

Comment:
When in "table mode" `&__description` & `&__aliases` should lose their `margin-top` (otherwise used for the vertical spacing between them) to be aligned with the label.
```suggestion
					margin-left: 16px;
					margin-top: 0;
```

Reply:
done